pkg/test/expectations/expectations.go

```diff
@@ -461,14 +463,33 @@ func ExpectOwnerReferenceExists(obj, owner client.Object) metav1.OwnerReference
 	return or
 }
 
+func ExpectFullyQualifiedNameFromCollector(collector prometheus.Collector) string {
```

Comment:
So this also has the added benefit of making sure that if a metric isn't found in the environment, we won't silently fail anymore, right?

Reply:
I wasn't considering that but you're right, it should.